ipa/ipa_provider.py

```diff
@@ -105,6 +106,7 @@ def __init__(self,
         self.host_key_fingerprint = None
         self.instance_ip = None
         self.provider = provider
+        self.collect_vm_info = collect_vm_info
```

Comment:
We should use the _get_value function here so that `collect_vm_info` can be in the config file as well.

```
self.collect_vm_info = self._get_value(
            collect_vm_info,
            config_key='collect_vm_info',
            default=False
        )
```

Reply:
thanks for the help I will modify my PR